src/python/espressomd/accumulators.py

```diff
@@ -3,8 +3,17 @@
 import numpy as np
 
 
+
+@script_interface_register
+class AccumulatorBase(ScriptInterfaceHelper):
```

Comment:
This should not be exposed, you can not even register `AccumulatorBase`, because it is abstract. If you want this on the python level just define an empty python class to derive from.